app/views/users/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1>Users</h1>
\ No newline at end of file
+<h1>Users</h1>
+<ul>
+  <% @users.each do |user| %>
+  <li>
+    <a href="<%= user_posts_path(user) %>">
+      <div class="user-box">
+        <%= render partial: 'shared/user_photo', locals: { user: user } %>
+        <%= render partial: 'shared/user_details', locals: { user: user } %>
+     </div>
```

Comment:
- [ ] Great work using partials to render commonly shared features across different pages. Great work 👏 💪